### DISCUSSION TITLE
Peer2profit

### DISCUSSION THREAD

**Discussion by hungf1511** [engageub/InternetIncome#65] [Q&A]
I found a project about running peer2profit on linux as a container. Can you bring peer2profit back?
-- https://hub.docker.com/r/enwaiax/peer2profit
-- https://github.com/pxzlz-ctrl/Peer2Profit

**Comment by engageub** [comment]
Hi,
If the docker containers are confirmed to be working, they can be added.
But the website seems to be available only on telegram as of now.

Thank you

**Reply by engageub** [reply]
Hi,
I don't have access to telegram now. If you can run the following and confirm that you see your device, the changes will be made to main branch.
You just have to enter email address in the following command. 
Replace <YOUR_EMAIL> with your email address.

```
sudo docker run --rm -it -e email=<YOUR_EMAIL> enwaiax/peer2profit
```

Thank you

**Comment by engageub** [comment]
It is confirmed to be working by other users.
Changes have been deployed to test branch.

https://github.com/engageub/InternetIncome/tree/test


Thank you

**Comment by engageub** [comment]
Changes have been deployed to main branch.
